modules/tiles/src/tileset/tile-3d.ts

```diff
@@ -303,6 +305,17 @@ export class Tile3D {
     return this._screenSpaceError;
   }
 
+  /**
+   * Get bounding box in cartographic coordinates
+   * @returns [min, max] each in [longitude, latitude, altitude]
+   */
+  get boundingBox(): [min: number[], max: number[]] {
+    if (!this._boundingBox) {
+      this._boundingBox = getCartographicBounds(this.header.boundingVolume, this.boundingVolume);
```

Comment:
To minimize perf impact, bounding box is only calculated on demand

Reply:
These are not needed by 3d tiles (hence we never calculated them before). Culling and rendering are done in the cartesian space.

Bounding boxes are regularly used in 2d tiles. The current implementation of `Tile2DHeader` in deck.gl takes a different format (`{west, east, south, north}` in geospatial viewports and `{left, right, bottom, top}` in non-geo viewports) which causes a lot of pains in typing. I am planning to converge the two.